html/includes/common/availability-map.inc.php

```diff
@@ -220,6 +235,32 @@
             '</select>
         </div>
         <div class="page-availability-title-right">';
+
+        $sql = 'SELECT `G`.`id`, `G`.`name` FROM `device_groups` AS `G`';
+        $dev_groups = dbFetchRows($sql);
```

Comment:
Maybe put the sql query inside the check for availability_map_use_device_groups check so it isn't executed every time.  Unless you need it.